frontends/web/src/routes/new-settings/components/settingsItem/settingsItem.tsx

```diff
@@ -18,27 +18,37 @@ import { ReactNode } from 'react';
 import styles from './settingsItem.module.css';
 
 type TProps = {
-    className?: string
-    onClick?: () => void;
-    settingName: string | ReactNode;
-    secondaryText?: string;
-    displayedValue?: string;
-    extraComponent?: ReactNode;
+  className?: string
+  collapseOnSmall?: boolean;
+  displayedValue?: string;
+  extraComponent?: ReactNode;
+  hideDisplayedValueOnSmall?: boolean;
+  onClick?: () => void;
+  secondaryText?: string;
+  settingName: string | ReactNode;
 }
 
 export const SettingsItem = ({
   className = '',
-  onClick,
-  settingName,
-  secondaryText,
+  collapseOnSmall = false,
   displayedValue = '',
   extraComponent,
+  hideDisplayedValueOnSmall = false,
```

Comment:
Had to add this since there's only one setting item that hides the displayed value on small screen, which is the `AttestationCheckSetting`. Other settings doesn't hide the displayed value.

On the screenshots below, displayed values are the encircled texts.

On desktop:
<img width="1002" alt="Screenshot 2023-05-23 at 12 04 55" src="https://github.com/digitalbitbox/bitbox-wallet-app/assets/28676406/1cea50ac-456b-434f-a2ad-fb48393bc050">

On mobile ("Your BitBox is authentic" gets hidden, but not other texts):
<img width="367" alt="Screenshot 2023-05-23 at 12 05 01" src="https://github.com/digitalbitbox/bitbox-wallet-app/assets/28676406/224dc4e2-7a65-4c62-af27-0b0aa7b593e4">